api.py

```diff
@@ -14,10 +16,10 @@ def error_handle(request_domain: str, expected_code: int, request_type: str, res
     :param response: response object
     """
     if response.status_code is not expected_code:
-        print(f'{request_type} request for {request_domain} failed with status code {response.status_code} '
+        logging.warning(f'{request_type} request for {request_domain} failed with status code {response.status_code} '
               f'(expected {expected_code}). Reason: {response.reason}')
         if response.status_code == 401:
-            print('NOTE: This may be because this is a new function, and additional authorization is required. '
+            logging.warning('NOTE: This may be because this is a new function, and additional authorization is required. '
                   'Try reauthorizing and try again.')
```

Comment:
This notice was added due to additional scopes being required in v1.1 update. This could probably be removed, otherwise it should be info.

Reply:
Removed in d3df488